project-news-site/code/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1-0">
+    <title>Project 2: The News Site</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Anton&display=swap" rel="stylesheet"> 
+  </head>
+
+  <!-- Navbar-->
+  <body>  
+    <header>
+      <div class="logo">Fat Cats Aficionado</div>
+      <div class="links">

```

Comment:
Instead of a div tag for the links, could a nav element maybe fitted better for that section?